__tests__/app.test.js

```diff
@@ -73,3 +73,27 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test("GET: 200, responds with an array of article objects, with the correct properties sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles[0].created_at).toBe("2020-11-03T09:12:00.000Z");
+        expect(body.articles[0].author).toBe("icellusedkars");
+        expect(body.articles[0].title).toBe(
+          "Eight pug gifs that remind me of mitch"
+        );
+        expect(body.articles[0].article_id).toBe(3);
+        expect(body.articles[0].topic).toBe("mitch");
+        expect(body.articles[0].votes).toBe(0);
+        expect(body.articles[0].article_img_url).toBe(
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        );
+        expect(Number(body.articles[0].comment_count)).toBe(2);
+        expect(!body.articles[0].body).toBe(true);
+        expect(body.articles).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+});
```

Comment:
This is a valid test but we want to ensure the entire array follows this example. Therefore, you can iterate over the array and your assertion can just check the data types for the properties. Have a look at `.toMatchObject` to help you with the assertion